scripts/management/slurm_query_status.py

```diff
@@ -145,17 +150,34 @@ def check_dependancy_met(task, task_list):
         return is_task_complete(LF_task, task_list) and is_task_complete(
             HF_task, task_list
         )
+
+    if process is Process.clean_up:
```

Comment:
Doesn't function 100% as desired but plot_flag should help with that.

Reply:
How should it function?

Reply:
If merge ts isn't complete it won't 'clean-up'. 
If we are skipping merge-ts (via no-merge-ts or plotting) then it should clean up by default unless no-clean up is set.